src/utilities/strings.ts

```diff
@@ -184,20 +184,39 @@ export const convertLinksToHTML = (string: string): string => {
     return ''
   }
 
-  const pattern = /(?:(https?:\/\/)|www\d{0,3}\.|www-|[a-z0-9.-]+\.[a-z]{2,4}(?=\/))(?:[^\s()<>]+)*(?:[^\s`!-()\[\]{};:'".,<>?«»“”‘’])/giu
-
-  return string.replace(
-    new RegExp(pattern),
-    (match: string, scheme): string => {
-      let url = match
-      if (!scheme) {
-        // Add http as the default scheme
-        url = `http://${url}`
+  const emailPattern = "\\b[A-Z0-9._'%+-]+@[A-Z0-9.-]+\\.[A-Z]{2,4}"
+  const urlPattern =
+    '(?:(?:https?:\\/\\/|www\\d{0,3}\\.|www-|[a-z0-9.-]+\\.[a-z]{2,4}(?=\\/))(?:[^\\s()<>]+)*(?:[^\\s`!-()\\[\\]{};:\'".,<>?«»“”‘’]))'
```

Comment:
Parsing strings for URLs and emails is hard. We took this simplified approach to reduce the bundle size in Beacon. It seems to do a pretty good job now, but there will be some edge cases.